src/NodaTime/LocalDate.cs

```diff
@@ -291,6 +291,7 @@ public static LocalDate FromYearMonthWeekAndDay(int year, int month, int occurre
 
         /// <summary>
         /// Adds the specified period to the date.
+        /// Fields are added in descending order of significance (years first, then months etc).
```

Comment:
nit: surely "months, etc.)."? (comma before, period after) (and throughout)

Reply:
I'll change to "and so on" to avoid the double period.